recipe/meta.yaml

```diff
@@ -103,122 +104,121 @@ outputs:
         - xpa {{ xpa }}.*
         - yaml-cpp {{ yaml_cpp }}.*
 
         ###################################################
         # run-like
-        - alembic >=1.14.1
-        - astropy-base >=7.0.1,<8
+        - alembic >=1.16.1
+        - astropy-base >=7.1.0,<8
         - astroplan >=0.10.1
         - astroquery ==0.4.9
-        - autograd >=1.7.0
+        - autograd >=1.8.0
         - backoff >=2.2.1
-        - batoid >=0.7.3
+        - batoid >=0.8.0
         - batoid-rubin >=0.5.0
-        - boto3 >=1.36.3
-        - botocore >=1.36.3
-        - bottleneck >=1.4.2
-        - cassandra-driver >=3.28.0  # [linux64]
-        - click >=8.1.7,<8.2
+        - boto3 >=1.39.11
+        - botocore >=1.39.11
+        - bottleneck >=1.6.0
+        - cassandra-driver >=3.29.2  # [linux64]
+        - click >=8.1.8,<8.2
         - coloredlogs >=15.0.1
         - colour-science >=0.4.6
-        - configparser >=7.1.0
-        - danish >=0.5.0
-        - deepdiff >=8.5.0
+        - configparser >=7.2.0
+        - danish >=0.6.0
+        - deepdiff >=8.6.1
         - defusedxml >=0.7.1
         - deprecated >=1.2.18
-        - documenteer >=2
-        - dustmaps >=1.0.13
+        - documenteer >=2.2.0
+        - dustmaps >=1.0.14
         - emcee >=3.1.6
-        - esutil >=0.6.11,!=0.6.15
-        - fastavro >=1.10.0
-        - firefly-client >=3.2.0
-        - fitsio >=1.2.5
+        - esutil >=0.6.16
+        - fastavro >=1.12.0
+        - firefly-client >=3.3.0
+        - fitsio >=1.2.7
         - fitsverify >=4.22
         - frozendict >=2.4.6
-        - fsspec >=2025.2.0
+        - fsspec >=2025.9.0
         - future >=1.0.0
         - gaiaxpy >=2.1.2
-        - galsim >=2.6.3
+        - galsim >=2.7.2
         - getcalspec >=2.1.0
-        - git >=2.47.1
-        - git-lfs >=3.6.1
-        - graphviz >=9.0.0
-        - h5py >=3.12.1
-        - healpy >=1.18.0
-        - healsparse >=1.10.0
+        - git >=2.51.0
+        - git-lfs >=3.7.0
+        - graphviz >=13.1.2
+        - h5py >=3.14.0
+        - healpy >=1.18.1
+        - healsparse >=1.11.1
         - hpgeom >=1.4.0
-        - htcondor >=24.0,<24.1
-        - humanize >=4.11.0
-        - idds-client >=2.1.37
-        - idds-doma >=2.1.37
-        - iminuit >=2.30.0
-        - jax >=0.4.35
-        - lmfit >=1.3,<1.4
+        - htcondor >=24.0.10,<24.1
+        - humanize >=4.13.0
+        - idds-client >=2.2.39
+        - idds-doma >=2.2.39
+        - iminuit >=2.31.1
+        - jax >=0.7.0
+        - lmfit >=1.3.4,<1.4
         - lsstdesc.coord >=1.3.0
-        - lsst-ts-xml >=22.1.0
+        - lsst-ts-xml >=23.3.2
         - matplotlib-base >=3.10
-        - maturin >=1.8.2
-        - moto >=5.0.28
-        - mpi4py >=4.0.2
+        - maturin >=1.9.4
+        - moto >=5.1.12
+        - mpi4py >=4.1.0
         - nest-asyncio >=1.6.0
-        - networkx >=3.4.2
+        - networkx >=3.5
         - ngmix-core >=2.4.0
         - numexpr >=2.10.2,<3
         - numpy
```

Comment:
should we have a numpy mini pin these days?

Reply:
Numpy is pinned above like this

```
        - numpy {{ numpy }}.*
```

on line 106. the second entry here is for completeness more than anything else. Leave things just as they are.